[2025-01-03T20:36:23.522-0800] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-03T20:36:23.561-0800] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: CovidE2EPipeDag.clean_up_sql_exe scheduled__2025-01-01T00:15:00+00:00 [queued]>
[2025-01-03T20:36:23.586-0800] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: CovidE2EPipeDag.clean_up_sql_exe scheduled__2025-01-01T00:15:00+00:00 [queued]>
[2025-01-03T20:36:23.588-0800] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-03T20:36:23.617-0800] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): clean_up_sql_exe> on 2025-01-01 00:15:00+00:00
[2025-01-03T20:36:23.623-0800] {standard_task_runner.py:72} INFO - Started process 67765 to run task
[2025-01-03T20:36:23.634-0800] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'CovidE2EPipeDag', 'clean_up_sql_exe', 'scheduled__2025-01-01T00:15:00+00:00', '--job-id', '406', '--raw', '--subdir', 'DAGS_FOLDER/CovidE2EPipeDag.py', '--cfg-path', '/var/folders/vm/rn9v1g1j739_mq8j5pfl1yxm0000gn/T/tmploffukkf']
[2025-01-03T20:36:23.638-0800] {standard_task_runner.py:105} INFO - Job 406: Subtask clean_up_sql_exe
[2025-01-03T20:36:23.721-0800] {task_command.py:467} INFO - Running <TaskInstance: CovidE2EPipeDag.clean_up_sql_exe scheduled__2025-01-01T00:15:00+00:00 [running]> on host Matthews-MacBook-Air.local
[2025-01-03T20:36:23.904-0800] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matthew Horn' AIRFLOW_CTX_DAG_ID='CovidE2EPipeDag' AIRFLOW_CTX_TASK_ID='clean_up_sql_exe' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T00:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T00:15:00+00:00'
[2025-01-03T20:36:23.907-0800] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-03T20:36:23.908-0800] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-03T20:36:23.909-0800] {logging_mixin.py:190} INFO - Current task name:clean_up_sql_exe state:running start_date:2025-01-04 04:36:23.562318+00:00
[2025-01-03T20:36:23.909-0800] {logging_mixin.py:190} INFO - Dag name:CovidE2EPipeDag and current dag run status:running
[2025-01-03T20:36:23.910-0800] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-03T20:36:23.916-0800] {subprocess.py:78} INFO - Tmp dir root location: /var/folders/vm/rn9v1g1j739_mq8j5pfl1yxm0000gn/T
[2025-01-03T20:36:23.917-0800] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'snow stage remove @raw_data_stage/load_staged_data_2025-01-03.sql;                   rm ~/airflow/helper_scripts/load_staged_data_2025-01-03.sql']
[2025-01-03T20:36:23.927-0800] {subprocess.py:99} INFO - Output:
[2025-01-03T20:36:29.701-0800] {subprocess.py:106} INFO - ╭─ Error ───────────────────────────────────╮
[2025-01-03T20:36:29.702-0800] {subprocess.py:106} INFO - │ Specified name                            │
[2025-01-03T20:36:29.703-0800] {subprocess.py:106} INFO - │ '@raw_data_stage/load_staged_data_2025-01 │
[2025-01-03T20:36:29.704-0800] {subprocess.py:106} INFO - │ -03.sql' is not valid name.               │
[2025-01-03T20:36:29.704-0800] {subprocess.py:106} INFO - ╰───────────────────────────────────────────╯
[2025-01-03T20:36:30.118-0800] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-03T20:36:30.155-0800] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-03T20:36:30.158-0800] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=CovidE2EPipeDag, task_id=clean_up_sql_exe, run_id=scheduled__2025-01-01T00:15:00+00:00, execution_date=20250101T001500, start_date=20250104T043623, end_date=20250104T043630
[2025-01-03T20:36:30.177-0800] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-03T20:36:30.179-0800] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-03T20:36:30.181-0800] {logging_mixin.py:190} INFO - Dag name:CovidE2EPipeDag queued_at:2025-01-04 04:33:43.805695+00:00
[2025-01-03T20:36:30.182-0800] {logging_mixin.py:190} INFO - Task hostname:Matthews-MacBook-Air.local operator:BashOperator
[2025-01-03T20:36:30.218-0800] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-03T20:36:30.247-0800] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-03T20:36:30.248-0800] {local_task_job_runner.py:245} INFO - ::endgroup::
